cmd/eksctl-anywhere/cmd/validations.go

```diff
@@ -33,3 +39,24 @@ func commonValidation(ctx context.Context, clusterConfigFile string) (*v1alpha1.
 	}
 	return clusterConfig, nil
 }
+
+func sourceValidation(source string) error {
```

Comment:
Also, it's more idiomatic to name functions that start with a verb[^1], ie validateSource or checkSource.

1: the exception is getter methods, they don't need get https://go.dev/doc/effective_go#Getters